### PR TITLE
Handle CallQueueTooBigException as a RecoverableException

### LINKED ISSUE #135: Retry "org.hbase.async.RemoteException: Call queue is full on" RPCs
HBase 1.x and later return an exception when the call queue is full. The native client will retry these calls as if it's was a recoverable exception. AsyncHBase should do the same.

### PR DESCRIPTION
**Pull request by stlava**
Fixes #135

### CODE REVIEW

File: src/CallQueueTooBigException.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+package org.hbase.async;
+
+/**
+ * Exception thrown when we try to use an invalid or expired scanner ID.
```
Comment (elliottneilclark) [inline src/CallQueueTooBigException.java:30]: Copy paste erro ?

Comment (stlava) [inline src/CallQueueTooBigException.java:30]: Yeah very much so copy pasta. Will update this and a formatting error I saw. I'm also verifying this fixes the issues as well.

**Comment by devaudio** [comment]
hmmm i still get:

org.hbase.async.CallQueueTooBigException: Call queue is full on /0.0.0.0:60020, too many items queued ?


when running tsdb uid metasync   

using this branch

**Comment by stlava** [comment]
This doesn't seem to address the issue. My understanding of how the recoverable exceptions are handled was wrong. I'll look further into this.

**Pull request closed without merging**